src/main/java/org/wso2/carbon/inbound/kafka/KafkaConstants.java

```diff
@@ -136,4 +138,14 @@ public class KafkaConstants {
 
     public static final String KAFKA_AVRO_DESERIALIZER = "io.confluent.kafka.serializers.KafkaAvroDeserializer";
 
+    // Error Handling Deserializer properties
+    public static final String KEY_DESERIALIZER_EXCEPTION_HEADER = "KafkaInboundKeyDeserializerException";
+    public static final String VALUE_DESERIALIZER_EXCEPTION_HEADER = "KafkaInboundValueDeserializerException";
+    public static final String DEFAULT_ERROR_HANDLING_DESERIALIZER_CLASS =
+            "org.wso2.carbon.inbound.kafka.deserializer.ErrorHandlingDeserializer";
+
+    // Error codes
+    public static final String POISON_PILL_DETECTED = "700601";
```

Comment:
Shall we use a different error code range? `7006xx` is used by the Snowflake connector

Reply:
What we need is not a Connector specific error code, rather Error type specific error code. For example, having a specific error code for POISON DETECTED for all connectors is better.

Reply:
Since all the connectors have maintained connector-specific error codes so far, without introducing a new code range, shall we use the existing Kafka connector error code range for this? WDYT?

Reply:
+1.

Reply:
Corrected by https://github.com/wso2-extensions/esb-inbound-kafka/pull/59/commits/ff12250e6c49f14d72ef658730c55f49849ce8a4